.drone.yml

```diff
@@ -1,55 +1,65 @@
-pipeline:
-  core:
-    group: tests
+kind: pipeline
+name: default
+steps:
```

Comment:
As far as I understand, "steps" will lead to each step to be executed sequentially. If so, why is the advantage "group"?

Reply:
Yes, so everything I said about the yml today is deprecated lol
This is a new format, and yes steps are usually sequencial but adding the `depemds_on=[clone] ` makes them parallel.

Reply:
depemds_on=[clone] was my next question. This new .yml format is so nonintuitive :)))

Reply:
Yeah, they made it worse.